CONTRIBUTING-TYPESCRIPT.md

```diff
@@ -15,18 +15,23 @@ This guide covers TypeScript-specific setup and development for AgentKit.
 
 ## Development Setup
 
-AgentKit uses Node.js v18.x or higher and npm 8.x or higher.
+AgentKit uses Node.js v18.x or higher and pnpm 10.7.x or higher.
 
-You can run the following commands in your terminal to check your local Node.js and npm versions:
+You can run the following commands in your terminal to check your local Node.js and pnpm versions:
 
 ```bash
 node --version
-npm --version
+pnpm --version
 ```
 
-If the versions are not correct or you don't have Node.js or npm installed, download through [nvm](https://github.com/nvm-sh/nvm).
+If the versions are not correct or you don't have Node.js or pnpm installed, download Node.Js through [fnm](https://github.com/Schniz/fnm) and then install [pnpm](https://pnpm.io/installation).
 
-Once you have these installed, make sure you install the project dependencies by running `npm install` from the root of the TypeScript monorepo, in other words, from the root `typescript/` folder.
+> **Note for existing contributors**: If you previously contributed when this was an npm turbo monorepo, you'll need to remove existing node_modules directories and reinstall dependencies with pnpm. From the `typescript/` directory, run:
+> ```bash
+> find . -name "node_modules" -type d -exec rm -rf {} + && pnpm i
+> ```
+
+Once you have these installed, make sure you install the project dependencies by running `pnpm install` from the root of the TypeScript monorepo, in other words, from the root `typescript/` folder.
```

Comment:
This line is not needed, the command above runs `pnpm i` already

Reply:
That note is only for contributors who were here before the migration. I think the test should still flow into install.

Perhaps I should move that note before the install step? Would that read better?

<img width="921" alt="Screenshot 2025-03-27 at 7 13 30 PM" src="https://github.com/user-attachments/assets/fe32f76f-0c72-49d2-b25c-b465de0dc7a4" />